src/securejoin/mod.rs

```diff
@@ -390,20 +401,20 @@ async fn send_handshake_msg(
     Ok(())
 }
 
-async fn chat_id_2_contact_id(context: &Context, contact_chat_id: ChatId) -> u32 {
-    if let [contact_id] = chat::get_chat_contacts(context, contact_chat_id).await[..] {
-        contact_id
+async fn chat_id_2_contact_id(context: &Context, contact_chat_id: ChatId) -> Result<u32, Error> {
```

Comment:
Ideally this should returns `Result<Option<u32>>` and `None` instead of `0`, but not in this PR.